strategy/from_json/date.ts

```diff
@@ -5,7 +5,7 @@ import { ERROR_INVALID_DATE } from "../../error_messages.ts";
 
 /** allows authors to pass a regex to parse as a date */
 export function createDateStrategy(regex: RegExp): FromJSONStrategy {
-  return function _createDateStrategy(input: JSONValue): Date {
+  return (input: JSONValue): Date => {
```

Comment:
I'm trying to remember if there was a benefit to having it as a named function before. Maybe how it would show up in stack traces?

Reply:
I think that is correct, this way `anon` function shows up. `createDateStrategy` will also be in the stack as the parent caller. It seems like the right time to use an arrow function after thinking about the use case. Thoughts?

Reply:
Yeah, that’s true, the parent function in the call trace will show up. So that seems fine.